testinfra/mon/test_ossec.py

```diff
@@ -20,7 +20,6 @@ def test_ossec_package(Package, package):
     assert Package(package).is_installed
 
 
-@pytest.mark.xfail(strict=True)
```

Comment:
Tests are failing due to #2442, where this was marked xfail - these will continue to fail until #2461 is addressed